tests/test_config.yml.example

```diff
@@ -0,0 +1,40 @@
+target-configs:
+  AndroidTarget:
+    entry-0:
+      timeout: 60
+      connection_settings:
+        device: 'emulator-5554'
```

Comment:
This structure still can't be built easily with simple templating. What we need is 
```
target-configs:
	entry-0:
	  AndroidTarget:
		  timeout: 60
		  connection_settings:
			device: 'emulator-5554'

	entry-1:
	  ...
```

Concretely we want to be able to easily build a config file using e.g. [jinja2](https://jinja.palletsprojects.com/en/3.1.x/) for loops. As it stands, you'd have to make jinja parse each config file you would be trying to merge, which is not possible. With the structure I'm recommending, this can be dealt by just including the content from multiple "snippet" files (and maybe use a filter to increase indentation level).

Reply:
Done.